src/ucp/core/ucp_am.c

```diff
@@ -359,6 +359,11 @@ ucp_am_fill_first_footer(ucp_am_first_ftr_t *ftr, ucp_request_t *req)
     ftr->total_size = req->send.length;
 }
 
+//Release user header buf back to UCP mpool
```

Comment:
This file changes are not part of the API request.
Specifically, the "API PR" usually contains only headers under `uc*/api/`